src/nvc.rs

```diff
@@ -29,6 +123,10 @@ pub struct Nvc {
     psw_nmi_pending: bool,
     psw_interrupt_mask_level: usize,
 
+    pub chcw_cache_enabled: bool,
+
+    pub cpu_cache: NvcCache,
```

Comment:
nit: I think just `cache` would be a better name than `cpu_cache` as the `Nvc` as it's modeled currently is already a cpu. In fact I think that's actually slightly incorrect now that I understand the system a bit better; I believe the actual Nvc consists of the CPU, timer, and some other glue logic, so calling the `Nvc` `V810` instead would probably make sense (but we can do that in a separate set of changes, just something I thought of here).